project/src/components/bookmark-button/bookmark-button.tsx

```diff
@@ -1,6 +1,17 @@
-function BookmarkButton(): JSX.Element {
+type BookmarkButtonProps = {
+  favorite: boolean;
```

Comment:
булев флаг, лучше используй 
```suggestion
  isFavorite: boolean;
```